docs/UserGuide.adoc

```diff
@@ -52,126 +54,141 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 * Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 ====
 
-=== Viewing help : `help`
+=== Queue
 
-Format: `help`
+==== Add a patient to the queue: `enqueue` ====
+Adds a patient to the queue +
+Format: `enqueue <PATIENT_REFERENCE_ID>`
 
-=== Adding a person: `add`
+==== Remove a patient from the queue: `dequeue` ====
+Removes a patient from the queue +
+Format: `dequeue <PATIENT_REFERENCE_ID>`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+==== Assign next patient to an available doctor : `next` ====
 
-[TIP]
-A person can have any number of tags (including 0)
+Assign next patient to an available doctor. Only used when a patient exits the room. +
+Format: `next`
 
-Examples:
+==== Doctor takes a break: `break`  ====
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+Doctor takes a break, the room becomes unavailable. +
+Format: `break break <ROOM_NUMBER>`
 
-=== Listing all persons : `list`
+==== Doctor resumes his/her duty: `resume` ====
+Doctor resumes his/her duty and can now serve patients. +
+Format: `resume <ROOM_NUMBER>`
 
-Shows a list of all persons in the address book. +
-Format: `list`
 
-=== Editing a person : `edit`
+=== Patient Management
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+==== Searches patient by arguments: `patient` ====
+Finds patients whose names contain any of the given keywords +
+Format: `patient <PATIENT_REFERENCE_ID> | <PATIENT_NAME> | <PHONE_NUM>`
 
-****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
-****
+==== Registers a new patient: `register` ====
+Registers a new patient +
+Format:  `register -name|-n <PATIENT_NAME> -ic|-i <PATIENT_IC> -hp <PHONE_NUM> -email|-e <EMAIL> [-address|-a <ADDRESS>] [-remark|-r <REMARK>]`
```

Comment:
please change to `register -name|-n <PATIENT_NAME> -id|-i <PATIENT_REFERENCE_ID> -phone|-p <PHONE_NUM> [-email|-e <EMAIL>] [-address|-a <ADDRESS>] [-remark|-r <REMARK>]`